gradle.properties

```diff
@@ -17,5 +19,6 @@ org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
 android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
-# Kotlin code style for this project: "official" or "obsolete":
-kotlin.code.style=official
\ No newline at end of file
+android.defaults.buildfeatures.buildconfig=true
+android.nonTransitiveRClass=false
+android.nonFinalResIds=false
```

Comment:
![memo-badge](https://img.shields.io/badge/review-memo-lightgrey) 

AGP Upgrade Assistant によるアップデート時に自動で追加されました。
既存のものに影響はなさそうだと感じたためそのままの記載としています。

Reply:
Flamingで新規プロジェクトを立ち上げた際は下記のみが追加されますね

```
# Enables namespacing of each library's R class so that its R class includes only the
# resources declared in the library itself and none from the library's dependencies,
# thereby reducing the size of the R class for that library
android.nonTransitiveRClass=true
```

Reply:
各種値は候補者がよしなに設定してもらうことを想定しても良いかもしれません

![スクリーンショット 2023-05-25 13 06 19](https://github.com/yumemi-inc/android-engineer-codecheck/assets/50792596/d6a71f70-b142-4e5c-8698-7759dd498c98)

参考：[Breaking changes: build option default values](https://developer.android.com/build/releases/gradle-plugin#default-changes)

Reply:
> 各種値は候補者がよしなに設定してもらうことを想定しても良いかもしれません

仰るとおり、候補者にお任せしたほうが良さそうですね。
https://github.com/yumemi-inc/android-engineer-codecheck/pull/25/commits/bd5655f1436660927546befe083ddfc8fd9480d0 にて項目をもとに戻しました。